OpenRA.Mods.Common/Traits/Player/ProvidesCustomPrerequisite.cs

```diff
@@ -34,12 +34,17 @@ public class ProvidesCustomPrerequisiteInfo : ITraitInfo
 	public class ProvidesCustomPrerequisite : ITechTreePrerequisite, INotifyOwnerChanged
 	{
 		readonly ProvidesCustomPrerequisiteInfo info;
+	    readonly string prerequisite;
```

Comment:
We don't have indention checks yet. It depends on https://github.com/shuruev/StyleCopPlus/pull/5.